dist/schema.json

```diff
@@ -27,6 +27,27 @@
         "responses": {
           "200": {
             "description": "[Token]",
+            "headers": {
+              "X-RateLimit-Limit": {
+                "description": "Number of requests permitted per time period",
+                "schema": {
+                  "type": "integer"
+                }
+              },
+              "X-RateLimit-Remaining": {
+                "description": "Number of requests you can still make",
+                "schema": {
+                  "type": "integer"
+                }
+              },
+              "X-RateLimit-Reset": {
```

Comment:
`X-RateLimit-Reset` should have a `type` of `string` and a `format` of `date-time`.